### PR TITLE
Load all modules after first build

### LINKED ISSUE #209: Issue with auto-completion when dialyzer is disabled for an umbrella app
### Environment
```
Erlang/OTP 21 [erts-10.3] [source] [64-bit] [smp:12:12] [ds:12:12:10] [async-threads:1] [hipe]
Elixir 1.7.3 (compiled with Erlang/OTP 21)
```
Language server version: v0.3.3

MacOS Mojave, Emacs with eglot as LS client

### Issue
Elixir LS fails to return autocompletion results for umbrella application when dialyzer is disabled

### Reproducing the issue
* clone https://github.com/akash-akya/elixir-ls-test
* open project with **dialyzer disabled**, with emacs and `eglot`, dialyzer is disabled by default using `.dir-locals.el`. 
* let elixir-ls build and generate .beam files
* open `apps/foo/lib/foo.ex` and type `Core.Bar` this should trigger auto-completion and it should show `Core.BarServer` as suggestion
* close the project and open it again
* try the same auto-completion as before, this time it won't show the suggestion

My best guess is, it is something to do with not triggering complete project compilation when we reopen the project.

### PR DESCRIPTION
**Pull request by akash-akya**
Closes #209 

The test contains _sleep_ as the build is async. I'm not sure about sleep duration. Let me know if there is a better way to test this

### CODE REVIEW

File: apps/language_server/test/server_test.exs [hunk 1]
```diff
@@ -336,4 +336,56 @@ defmodule ElixirLS.LanguageServer.ServerTest do
...
+
+  test "loading of umbrella app dependencies", %{server: server} do
+    in_fixture(__DIR__, "umbrella", fn ->
+      # We test this by opening the umbrella project twice.
+      # First to compile the applications and build the cache.
+      # Second time to see if loads modules
+      initialize(server)
+
+      # Upon first vist, server complies and loads all umbrella applications and modules
+      Process.sleep(1_500)
```
Comment (lukaszsamson) [inline apps/language_server/test/server_test.exs:348]: If the server process is handling a message, then simply calling `:sys.get_state` will make the calling process block until the server process is ready. If the server is waiting for an async message from other process you can call `:sys.get_state` in a loop with short sleeps until the state returned indicates that async task is completed.

Comment (akash-akya) [reply apps/language_server/test/server_test.exs:348]: ok. I'll will try to update test

Comment (akash-akya) [reply apps/language_server/test/server_test.exs]: updated, please check

Comment (lukaszsamson) [reply apps/language_server/test/server_test.exs]: Looks good

**Pull request merged**